packages/transition-frontend/src/components/forms/comparison/ScenarioComparisonPanel.tsx

```diff
@@ -305,13 +308,25 @@ class ScenarioComparisonPanel extends ChangeEventsForm<ComparisonPanelProps, Com
         return '';
     }
 
+    private updateBothRoutingEngines = (
+        path: string,
+        newValue: { value: any; valid?: boolean },
+        altRouting: TransitRouting,
+        resetResults = true
+    ) => {
+        this.onValueChange(path, newValue, resetResults);
+        if (newValue.valid || newValue.valid === undefined) {
+            altRouting.attributes[path] = newValue.value;
+        }
+    };
+
     render() {
         if (!this.state.scenarioCollection) {
             return <LoadingPage />;
         }
 
         const routing = this.state.object;
-        const alternateRouting = this.state.alternateScenarioRouting;
+        const altRouting = this.state.alternateScenarioRouting;
```

Comment:
Why reduce the variable name ? Its usually better to keep full names, character space is cheap thse days :)